float32 x
float32 y
float32 z
float32 yaw
string frame_id
bool auto_arm
---
bool success
string message
